pkg/cli/job/delete.go

```diff
@@ -35,13 +35,30 @@ type deleteFlags struct {
 	JobName   string
 }
 
+func (dflags *deleteFlags) GetMasterUrl() string {
+	return dflags.Master
+}
+
+func (dflags *deleteFlags) GetKubeconfigPath() string {
+	return dflags.Kubeconfig
+}
+
+func (dflags *deleteFlags) GetNamespace() string {
+	return dflags.Namespace
+}
+
+func (dflags *deleteFlags) SetNamespace(ns string) error {
+	dflags.Namespace = ns
+	return nil
+}
+
```

Comment:
There are many similar repetitive functions below, I think this kind of scene is very suitable for generics